### ISSUE TITLE
Not showing window resize controls on some windows if you hover the edge of a window with the cursor

### ISSUE THREAD

**Issue by tristan957** [solus-project/budgie-desktop#535]
Noticed this was happening on Nautilus. I can resize the window if I click the edge. This behavior doesn't happen on GNOME Terminal however. I get the resize controls on mouse hover. I will try and report back with other programs in a bit.

**Comment by JannikHa** [comment]
Which Budgie-Version are you using? Tried to repoduce the problem (Budgie V10.2.7) but failed.

I'm not sure if i understand you problem *exactly*. So when you hover over the edge of some windows, you **can** resize it but the **resize controls** (your cursor) are not shown, is that correct?

**Comment by tristan957** [comment]
What you described it's exactly my problem. Not sure if this has anything to do with it, but hovering the text input field in gedit does not change the cursor to the vertical bar. Another application that does this is system monitor, regarding the resize controls. On Solus 10.2.7

**Comment by JannikHa** [comment]
This is very weird. This question may seem odd, but what **cursor Theme** and GTK theme in general are you using?

**Comment by tristan957** [comment]
Solus defaults. Breeze Snow and Arc Darker

**Comment by tristan957** [comment]
Solus defaults. Breeze Snow and Arc Darker
On Oct 18, 2016 7:22 AM, Jannik Hauptvogel <notifications@github.com> wrote:This is very weird. This question may seem odd, but what cursor Theme and GTK theme in general are you using?

**Comment by JannikHa** [comment]
What is the output of:

    ls /usr/share/icons/
    ls /usr/share/icons/Adwaita/
?
Some Cursor themes use Adwaita as kind of "back-end" for certein actions (like resizing for example) . Slowly but surely I'm running out of ideas i must admit, this problem is really strange.

**Comment by tristan957** [comment]
I get the window resize controls for Chrome and Atom. It seems to only be GTK 3 apps with a headerbar.  The reason I have come to this conclusion is that GNOME Terminal will display the window resize controls.

**Comment by JannikHa** [comment]
You should try using a **different cursor theme**, the best would be the **Adwaita** one. Reboot after selecting it and try to replicate your problem. I don't know but I'm actually sure it's the theme's problem to be honest...

**Comment by ikeydoherty** [comment]
I've come to the conclusion it may be an Arc issue actually.. try with a different GTK+ theme :P Reboot not needed for GTK themes though, on the CSD GNOME 3 apps

**Comment by horst3180** [comment]
@ikeydoherty Nope, this happened with every theme I tested (Adwaita, Adapta, Menta, Paper, Vertex). Also, the resize cursor actually appears when you press the mouse button over the resize area.

**Comment by ikeydoherty** [comment]
@horst3180 i thought it was gonna be like one of those weird bugs we had with old components + Arc before (Like that librsvg issue) :( 

So, GTK issue itself?

**Comment by horst3180** [comment]
@ikeydoherty Probably. The same happens under MATE. That rules out mutter at least.

**Comment by ikeydoherty** [comment]
Aw but mutter is my usual point of blame XD

**Comment by tristan957** [comment]
Nice to see that I'm not the only one with this issue :D
